src/Package/Impl/Help/VignetteCodeColorBuilder.cs

```diff
@@ -0,0 +1,96 @@
+            new CssCodeProperty("keyword", PredefinedClassificationTypeNames.Keyword),
+            new CssCodeProperty("number", PredefinedClassificationTypeNames.Number),
+            new CssCodeProperty("string", PredefinedClassificationTypeNames.String),
+            new CssCodeProperty("literal", PredefinedClassificationTypeNames.Literal),
+            new CssCodeProperty("comment", PredefinedClassificationTypeNames.Comment),
+            new CssCodeProperty("identifier", PredefinedClassificationTypeNames.Identifier),
+            new CssCodeProperty("operator", PredefinedClassificationTypeNames.Operator),
+            new CssCodeProperty("paren", PredefinedClassificationTypeNames.Operator),
+        };
+
+        private readonly IRInteractiveWorkflowProvider _workflowProvider;
+        private readonly IClassificationFormatMapService _formatMapService;
+        private readonly IClassificationTypeRegistryService _classificationRegistryService;
+
+        [ImportingConstructor]
+        public VignetteCodeColorBuilder(
+            IRInteractiveWorkflowProvider workflowProvider,
+            IClassificationFormatMapService cfms, 
+            IClassificationTypeRegistryService ctrs) {
+            _workflowProvider = workflowProvider;
+            _formatMapService = cfms;
+            _classificationRegistryService = ctrs;
+        }
+
+        public string GetCodeColorsCss() {
+            var sb = new StringBuilder();
+            var workflow = _workflowProvider.GetOrCreate();
+            if (workflow.ActiveWindow?.TextView != null) {
+                var map = _formatMapService.GetClassificationFormatMap(workflow.ActiveWindow.TextView);
+                foreach (var cssCodeProp in _cssPropertyMap) {
+                    var props = map.GetTextProperties(_classificationRegistryService.GetClassificationType(cssCodeProp.ClassificationTypeName));
+
+                    sb.AppendLine(Invariant($"code > span.{cssCodeProp.CssClassName} {{"));
+                    sb.AppendLine(Invariant($"color: {CssColorFromBrush(props.ForegroundBrush)};"));
+                    sb.AppendLine("}");
+                    sb.AppendLine(string.Empty);
+                }
+                foreach (var cssCodeProp in _prePropertyMap) {
+                    var props = map.GetTextProperties(_classificationRegistryService.GetClassificationType(cssCodeProp.ClassificationTypeName));
+
+                    sb.AppendLine(Invariant($"pre .{cssCodeProp.CssClassName} {{"));
+                    sb.AppendLine(Invariant($"color: {CssColorFromBrush(props.ForegroundBrush)};"));
+                    sb.AppendLine("}");
+                    sb.AppendLine(string.Empty);
+                }
+            }
+            return sb.ToString();
+        }
+
+        private string CssColorFromBrush(System.Windows.Media.Brush brush) {
+            var sb = brush as System.Windows.Media.SolidColorBrush;
```

Comment:
It conflicts with System.Drawing unfortunately. Help window is Windows.Forms browser